ddl/job_table.go

```diff
@@ -255,6 +274,7 @@ func (d *ddl) delivery2worker(wk *worker, pool *workerPool, job *model.Job) {
 			// If the job is done or still running or rolling back, we will wait 2 * lease time to guarantee other servers to update
 			// the newest schema.
 			waitSchemaChanged(context.Background(), d.ddlCtx, d.lease*2, schemaVer, job)
```

Comment:
same to 231, make sure it will not block a long period for woker

Reply:
It's ok here for the disabled MDL case.